src/stretchy.js

```diff
@@ -453,7 +461,7 @@ const svgSpan = function(group, options) {
     if (utils.contains(["widehat", "widetilde", "undertilde"], label)) {
         // There are four SVG images available for each function.
         // Choose a taller image when there are more characters.
-        const numChars = group.value.value.length;
+        const numChars = groupLength(group.value.base);
```

Comment:
Stretchy accents use `body` instead of base.  Probably best to make them use the same thing.  I have a slight preference for `body` because `base` makes me think of exponents, but either would be fine.